.github/workflows/changelog.yml

```
@@ -77,7 +77,13 @@ jobs:
       # In order to validate any links in the yaml file, render the config to markdown
       - name: Render .chloggen changelog entries
         run: |
-          make chlog-preview 2> changelog_preview.md
-          cat changelog_preview.md
-      - name: Run markdown-link-check
-        run: make markdown-link-check-changelog-preview
+          mkdir -p ${{ github.workspace }}/tmp
+          make chlog-preview &> ${{ github.workspace }}/tmp/changelog_preview.md
```

is there problem with using `/tmp/`?
```suggestion
          make chlog-preview &> /tmp/changelog_preview.md
```

---

not really, just more complicated docker command with multiple targets (to config and to temp). I came back to relative path - I changed it while going into a different rabbit hole